app/components/overview-info-form/component.js

```diff
@@ -28,61 +30,69 @@ var generateValidators = function(questions) {
 
 const questions = {
   q1: {
-    question: 'Age',
+    question: translations.survey.sections['1'].questions['1'].label,
```

Comment:
I am iterating over `questions`, so that is what needs to be a list, right? Everything else I am accessing directly